libs/locales/package.json

```diff
@@ -51,5 +51,5 @@
     "lint": "yarn run -T eslint --cache --cache-location node_modules/.cache/eslint/.eslint-cache ."
   },
   "type": "module",
-  "version": "1.0.0"
```

Comment:
Do we usually update the versions of our libs? Isn't that done automatically during the releases?

Reply:
@jgyselov the base branch was incorrect. Changed it